File: packages/core/test/fixtures/regex/java/launchdarkly/positive/basic.java
import com.launchdarkly.sdk.server.LDClient;

public class Checkout {
    private final LDClient client;

    public String run(LDContext context) {
        boolean enabled = client.boolVariation("CHECKOUT_V2", context, false);
        return enabled ? "v2" : "v1";
    }
}

File: packages/core/test/fixtures/regex/kotlin/launchdarkly/positive/basic.kt
import com.launchdarkly.sdk.server.LDClient

class Checkout(private val client: LDClient) {
    fun run(context: LDContext): String {
        val enabled = client.boolVariation("CHECKOUT_V2", context, false)
        if (enabled) {
            return "v2"
        }
        return "v1"
    }
}
